README.md

```diff
@@ -34,6 +34,32 @@ The following reference pages are available to describe the Azure App Configurat
   * [Fiddler](./docs/REST/fiddler.md)
   * [Postman](./docs/REST/postman.md)
 
+## Source
+
+### Dotnet Provider
+
+Module | Version | Sample
+-------|--- | ---
+DotnetProvider | [![NuGet](https://img.shields.io/nuget/v/Microsoft.Azure.AppConfiguration.AspNetCore.svg?color=blue)](https://www.nuget.org/packages/Microsoft.Azure.AppConfiguration.AspNetCore/) | [Sample](https://github.com/Azure/Azconfig-DotnetProvider/tree/master/examples)
```

Comment:
We have two Nuget packages available for .NET.
.NET Standard : Microsoft.Extensions.Configuration.AzureAppConfiguration
ASP.NET Core : Microsoft.Azure.AppConfiguration.AspNetCore

Reply:
This reminded me to include the config provider for .NET Framework as well

Package: https://www.nuget.org/packages/Microsoft.Configuration.ConfigurationBuilders.AzureAppConfiguration

Source:
https://github.com/aspnet/MicrosoftConfigurationBuilders/tree/master/src/AzureAppConfig

Maybe we should add a separate column in the table, for example, "Platform" and it can be ".NET Standard", ".NET Framework", "ASP.NET Core", "Spring Boot", "Spring Boot Web" etc. 

Thoughts?

Reply:
... and another column for "Release Notes".

Reply:
I think I got all of this.